.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/php:7.3-node-browsers
```

Comment:
don't wanna do 7.4?

Reply:
I made it PHP 7.3 since it's lib that may be used by non PHP 7.4 apps.